variables.tf

```diff
@@ -215,3 +215,8 @@ variable "cluster_name_random_string" {
   description = "Add a random string to the cluster name"
   default     = false
 }
+
+variable "subnet_range" {
+  description = "Private IP space to be used in CIDR format"
+  default     = ""
```

Comment:
i think this needs to clone the default. Rebasing to v0.2 @geekbass can you change the default here

Reply:
This is now resolved with the recent changes that Weston made